besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -629,9 +629,8 @@ void setBannedNodeIds(final List<String> values) {
   @Option(
       names = {"--logging", "-l"},
       paramLabel = "<LOG VERBOSITY LEVEL>",
-      description =
-          "Logging verbosity levels: OFF, FATAL, ERROR, WARN, INFO, DEBUG, TRACE, ALL (default: ${DEFAULT-VALUE})")
-  private final Level logLevel = LogManager.getRootLogger().getLevel();
+      description = "Logging verbosity levels: OFF, FATAL, ERROR, WARN, INFO, DEBUG, TRACE, ALL")
+  private final Level logLevel = null;
```

Comment:
```suggestion
  private final Level logLevel;
```
The setting to `null` should be implicit, no?

Reply:
Ignore ....`final` modifier is consistent with class